<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddItemDialogue</class>
 <widget class="QDialog" name="AddItemDialogue">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Add Item</string>
  </property>
  <widget class="QLabel" name="lblName">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Product Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtProductName">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>60</y>
     <width>201</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblQuantity">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Quantity</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="sbQuantity">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>100</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>999</number>
   </property>
  </widget>
  <widget class="QLabel" name="lbl">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>50</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Image:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblImage">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>120</y>
     <width>250</width>
     <height>250</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::Panel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnConfirmAdd">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>420</y>
     <width>301</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirm Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLoadImage">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>80</y>
     <width>141</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Load Item Image</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblBrand">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Brand</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtBrand">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>140</y>
     <width>171</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblBrand_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>180</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Size</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblBrand_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>220</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Category</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblBrand_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>260</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Deposit</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtSize">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>180</y>
     <width>171</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtCategory">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>220</y>
     <width>171</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtDeposit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>260</y>
     <width>171</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="sbMinimumStock">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>300</y>
     <width>71</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblBrand_5">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>300</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Min:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblName_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Add Item</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
